directives/thebe.py

```diff
@@ -0,0 +1,252 @@
+"""A sphinx extension to enable interactive computations using thebe."""
```

Comment:
I will need to add the license and copyright here since this code is adapted from the Thebe project.
https://github.com/executablebooks/sphinx-thebe/blob/master/LICENSE